site/en/docs/devtools/console/api/index.md

```diff
@@ -3,14 +3,20 @@ layout: "layouts/doc-post.njk"
 title: "Console API reference"
 authors:
   - kaycebasques
+  - sofiayem
 date: 2016-03-21
-#updated: YYYY-MM-DD
+updated: 2022-07-22
 description: "Use the Console API to write messages to the Console."
 ---
 
```

Comment:
Include the video url here.

Reply:
Will update when the video is out. But I think we can merge the refresh ahead of time.